.github/workflows/ci.yml

```
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Any reason to not pick a more recent version?

```suggestion
          node-version: 22
```

---

No reason at all, I just used the same as was being used in line 25

---

Maybe change it there too then

---

done https://github.com/storyblok/mui/pull/46/commits/870420db08c7cb4fa828f93693930e4f68e12672!